2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_setup.py:_flush():75] Loading settings from /home/atharva1511/.config/wandb/settings
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_setup.py:_flush():75] Loading settings from /home/atharva1511/Downloads/CS6910/assignment2/code2/MLFFN/wandb/settings
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Downloads/CS6910/assignment2/code2/MLFFN/train.py', 'program': 'train.py'}
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_init.py:_log_setup():405] Logging user logs to /home/atharva1511/Downloads/CS6910/assignment2/code2/MLFFN/wandb/run-20220401_112219-x89ggo1u/logs/debug.log
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_init.py:_log_setup():406] Logging internal logs to /home/atharva1511/Downloads/CS6910/assignment2/code2/MLFFN/wandb/run-20220401_112219-x89ggo1u/logs/debug-internal.log
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_init.py:init():439] calling init triggers
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'checkpoint_dir': 'models', 'train_file_path': '../encoder_compress.csv', 'val_file_path': '../encoder_compress.csv', 'cuda': False, 'wandb': False, 'do_train': True, 'do_val': True, 'do_predict': True, 'use_trained': True, 'log_step': 100, 'batch_size': 64, 'epochs': 100, 'val_batch_size': 64, 'output_num': 5, 'input_num': 10, 'num_hidden1': 25, 'num_hidden2': 19, 'learning_rate': 0.001, 'momentum': 0.9198, 'weight_decay': 0.0}
2022-04-01 11:22:19,608 INFO    MainThread:7953 [wandb_init.py:init():492] starting backend
2022-04-01 11:22:19,608 INFO    MainThread:7953 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-01 11:22:19,612 INFO    MainThread:7953 [backend.py:ensure_launched():219] starting backend process...
2022-04-01 11:22:19,617 INFO    MainThread:7953 [backend.py:ensure_launched():224] started backend process with pid: 7975
2022-04-01 11:22:19,618 INFO    MainThread:7953 [wandb_init.py:init():501] backend started and connected
2022-04-01 11:22:19,621 INFO    MainThread:7953 [wandb_init.py:init():565] updated telemetry
2022-04-01 11:22:19,623 INFO    MainThread:7953 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-01 11:22:20,636 INFO    MainThread:7953 [wandb_run.py:_on_init():1759] communicating current version
2022-04-01 11:22:20,771 INFO    MainThread:7953 [wandb_run.py:_on_init():1763] got version response 
2022-04-01 11:22:20,771 INFO    MainThread:7953 [wandb_init.py:init():625] starting run threads in backend
2022-04-01 11:22:21,317 INFO    MainThread:7953 [wandb_run.py:_console_start():1733] atexit reg
2022-04-01 11:22:21,318 INFO    MainThread:7953 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-04-01 11:22:21,318 INFO    MainThread:7953 [wandb_run.py:_redirect():1611] Redirecting console.
2022-04-01 11:22:21,319 INFO    MainThread:7953 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-01 11:22:21,319 INFO    MainThread:7953 [wandb_init.py:init():664] run started, returning control to user process
2022-04-01 11:25:04,057 INFO    MainThread:7953 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-01 11:25:04,059 INFO    MainThread:7953 [wandb_run.py:_restore():1674] restore
2022-04-01 11:25:11,622 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:16,723 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:21,825 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:26,926 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:32,028 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:37,129 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:42,230 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:47,331 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:52,433 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:25:57,534 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:02,635 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:07,736 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:12,837 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:17,938 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:23,039 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:28,141 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:33,242 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:38,343 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:43,444 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:48,545 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:53,646 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:26:58,747 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:27:03,849 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:27:08,950 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:27:14,051 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
2022-04-01 11:27:19,153 INFO    MainThread:7953 [wandb_run.py:_on_finish():1831] got exit ret: None
